e2e-tests/commands/save-session.js

```diff
@@ -1,11 +1,40 @@
-/**
- * Since cookies are cleared before individual tests execution.
- * Preserving the cookie, eliminates repeated logins.
- * Thus reduces execution time.
- */
-const saveSession = () => cy.preserveCookieOnce(
-  'exip-session', // Session cookie
-  '_csrf', // CSRF cookie
-);
+import { COOKIE } from '../constants';
+
+const cookies = [
+  COOKIE.NAME.SESSION,
+  COOKIE.NAME.CSRF,
+];
+
+const flags = {
+  secure: true,
+  httpOnly: true,
+  path: '/',
+  log: false,
+};
+
+const saveSession = () => {
```

Comment:
Could we add some documentation and maybe link to/mention "based on preserve cookie package"?